dropwizard-configuration/src/main/java/io/dropwizard/configuration/ResourceConfigurationSourceProvider.java

```diff
@@ -6,6 +6,11 @@
 public class ResourceConfigurationSourceProvider implements ConfigurationSourceProvider {
     @Override
     public InputStream open(String path) throws IOException {
+        InputStream result = getResourceAsStream(path);
+        return result == null && path.startsWith("/") ? getResourceAsStream(path.substring(1)) : result;
```

Comment:
I think an inline comment is warranted here, pointing out the convention of implicitly treating absolute resource paths as being rooted within the application's resources directory [1]. 

[1] https://docs.oracle.com/javase/8/docs/technotes/guides/lang/resources.html#res_names